app/backend/src/tests/test_api.py

```diff
@@ -162,9 +162,18 @@ def test_update_profile(db):
         assert user.lat == 0.01
         assert user.lng == -2
         assert user.radius == 321
+        assert user.gender == "Bot"
+        assert user.occupation == "Testing"
+        assert user.about_me == "I rule"
+        assert user.about_place == "My place"
+        assert user.color == "#111111"
```

Comment:
We're taking out color in the #410

Reply:
Oh, thanks for noticing, and thanks again for fixing it in this branch :) Merging!